chameleon.cfg

```diff
@@ -4,15 +4,29 @@ parts += chameleon-cache
 environment-vars +=
     CHAMELEON_EAGER ${buildout:chameleon-eager}
     CHAMELEON_RELOAD ${buildout:chameleon-reload}
-    CHAMELEON_CACHE ${buildout:chameleon-cache}
     FTW_CHAMELEON_RECOOK_WARNING ${buildout:chameleon-recook-warning}
     FTW_CHAMELEON_RECOOK_EXCEPTION ${buildout:chameleon-recook-exception}
 
 
 [chameleon-cache]
 recipe = collective.recipe.shelloutput
 commands =
-    cmd1 = mkdir -p ${buildout:chameleon-cache}
+    cmd = mkdir -p ${buildout:directory}/var/instance0/chameleon-cache
```

Comment:
Hmm. This list seems to be lacking `var/instance/chameleon-cache` (for local Plone development).

Didn't creating only the required directories as needed work yesterday?